src/commands/openInPortal.ts

```diff
@@ -16,7 +17,10 @@ export async function openInPortal(context: IActionContext, node?: AzExtTreeItem
     if (node instanceof ContainerAppTreeItem) {
         // ContainerApp's id don't include the ManagedEnvironment (ie the parent) so don't use fullId
         await azUtil.openInPortal(node, <string>node.data.id);
-    } else {
+    } else if (node instanceof LogsTreeItem) {
+        await azUtil.openInPortal(node, `${node.parent.id}/logs`)
```

Comment:
I think it would be better to appropriately set each tree item's id in the tree item class rather than add special cases here

Reply:
Some context-- a lot of azure resource IDs build on each other (such as web app/slots or static app/environment build) so we were using full id for most of those cases.  Container Apps only has 3 resources to open in portal and container apps do not include their environment as part of the Azure ID.  Anyway, the ids were already properly configured to work in the portal so this change should be good